Traceback (most recent call last):
  File "d:\Projects\temp\vorticity_MDNN_dg.py", line 72, in <module>
    pinn_nvs = train_vorticity_dg(config,device=device)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\Projects\temp\nv_files\train_nvs.py", line 482, in train_vorticity_dg
    batch = next(samples_interior).to(device)
            ^^^^^^^^^^^^^^^^^^^^^^
  File "d:\Projects\temp\nv_files\data_generator.py", line 80, in __getitem__
    batch = self.data_generation()
            ^^^^^^^^^^^^^^^^^^^^^^
  File "d:\Projects\temp\nv_files\data_generator.py", line 114, in data_generation
    batch = min_vals + rand_vals * (max_vals - min_vals)
                       ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~
RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cuda:0 and cpu!
